pysnooper/tracer.py

```diff
@@ -182,7 +188,7 @@ def __init__(
              v if isinstance(v, BaseVariable) else Exploding(v)
              for v in utils.ensure_tuple(watch_explode)
         ]
-        self.frame_to_local_reprs = collections.defaultdict(lambda: {})
+        self.frame_to_local_reprs = collections.OrderedDict()
```

Comment:
```suggestion
        self.frame_to_local_reprs = {}
```

I agree there's not much use to a defaultdict here (we'll see what @cool-RR thinks) but there's no reason for this to be ordered.

Reply:
Agreed there's no need for `defaultdict`. I'll let you guys figure out whether an `OrderedDict` is necessary or not, I don't mind.

Reply:
OK, remove the ordereddict. we don't care about the order of frames.